src/utils/traverse.js

```diff
@@ -102,6 +103,7 @@ const ORDER = {
   LogicalNotOp: ['expression'],
   LogicalOrOp: ['left', 'right'],
   MemberAccessOp: ['expression'],
+  MultiplyOp: ['left', 'right'],
```

Comment:
No, this fixes the problem with having the "multiply" operator. I didn't want to open 4 different pull requests to simplify things a bit (for me and for you), but I can if you prefer. It fixes the folowing test : https://github.com/eventualbuddha/decaffeinate/pull/40/files#diff-d2b10ceae8b94887ff3288766d636a9fR455